core/vm/stack.go

```diff
@@ -5,16 +5,20 @@ import (
 	"math/big"
 )
 
-func newStack() *stack {
-	return &stack{}
+func newStack() *Stack {
+	return &Stack{}
 }
 
-type stack struct {
+type Stack struct {
```

Comment:
Because it was passed outside of the VM. But in a later iteration of the structure logging I removed it. This will change back